src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -8903,7 +8906,7 @@ bool Unit::IsInDisallowedMountForm() const
     ChrRacesEntry const* race = sChrRacesStore.LookupEntry(displayExtra->Race);
 
     if (model && !(model->Flags & 0x80))
-        if (race && !(race->Flags & 0x4))
+        if (race && !(race->HasFlag(ChrRacesFlags::CHRRACES_FLAGS_CAN_MOUNT)))
```

Comment:
this change is present in both of your PRs

Reply:
Oooh, yea it just looks that way because they're two similar flags. Information taken from wowdev wiki. Hopefully they are right this time hehe.